import_tracker/web_client/templates/importList.pug

```diff
@@ -80,9 +80,9 @@ table.g-imports-list-table
           span= _import.get('_assetstoreName')
         if anyImportPath
           td(data-id=_import.get('params').importPath, data-toggle='tooltip', title=_import.get('params').importPath)
-            span= _import.get('params').importPath
+            span= (_import.get('params').importPath || '').replaceAll('/', '/\u00AD')
```

Comment:
I would like to mention that this linebreak regex sometimes leaves a leading `/` in some cases. Not sure if this is desired
![image](https://github.com/DigitalSlideArchive/import-tracker/assets/54944980/0491d707-836b-4b1e-8c49-1d725521ae4c)

Reply:
Not brilliant, but probably still better than not breaking at all.  We could only add the soft hyphen after all but the first / 
 (commit to follow).

Reply:
Great! Looking good